mantidproject/mantid
# Move JumpFit from Bayes to IDA

Fixes #12985.

To test:
- Open Indirect > Data Analysis > JumpFit
- Load a file (`babylon5/Public/DanNixon/data/cf_Result.nxs`)
- See that the range selectors work as expected

A lot of the interface code has poor indentation formatting so may as well just run clang-format to fix it.

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24432&oldid=24419).
